Release/src/http/client/x509_cert_utilities.cpp

```diff
@@ -375,55 +370,67 @@ bool verify_X509_cert_chain(const std::vector<std::string>& certChain, const std
 #endif
 
 #if defined(_WIN32)
-namespace
-{
-// Helper RAII unique_ptrs to free Windows structures.
-struct cert_free_certificate_context
-{
-    void operator()(const CERT_CONTEXT* ctx) const { CertFreeCertificateContext(ctx); }
-};
-typedef std::unique_ptr<const CERT_CONTEXT, cert_free_certificate_context> cert_context;
-struct cert_free_certificate_chain
-{
-    void operator()(const CERT_CHAIN_CONTEXT* chain) const { CertFreeCertificateChain(chain); }
-};
-typedef std::unique_ptr<const CERT_CHAIN_CONTEXT, cert_free_certificate_chain> chain_context;
-}
-
-bool verify_X509_cert_chain(const std::vector<std::string>& certChain, const std::string&)
+bool verify_X509_cert_chain(const std::vector<std::string>& certChain, const std::string& hostname)
```

Comment:
This can be `static`

Reply:
It's declared static at the top of the file.